test/commands/logic/run/test.nut.ts

```diff
@@ -0,0 +1,206 @@
+/*
+ * Copyright 2025, Salesforce, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import path from 'node:path';
+import fs from 'node:fs';
+import { execCmd, TestSession } from '@salesforce/cli-plugins-testkit';
+import { expect, config } from 'chai';
+import { TestRunIdResult } from '@salesforce/apex-node/lib/src/tests/types.js';
+import { RunResult } from '../../../../src/reporters/index.js';
+
+config.truncateThreshold = 0;
+
+describe('logic run test', () => {
+  let session: TestSession;
+  before(async () => {
```

Comment:
if it's the same as the setup for `logic get test` NUTs can we consolidate them together?